Update quick-cd-tips.md (#2465)
enkidevs/curriculum: linux/user-and-file-management/shell/quick-cd-tips.md

The `pushd` command saves the current working directory in memory so it can be returned to at any time. You can think of it as an evolution of `cd`, because `pushd` stores the history of your visited folders.

These visited folders/directories are saved in what's called "the directory stack".

To move to a new directory, you would use:

```bash
pushd projects/
~/projects ~
```

After calling `pushd` followed by a folder name, the directory stack is logged in the shell and you're switched to that folder.

> > Tilde `~` is a short hand notation for `Users/your-name/`.

Reviewer: ```suggestion
> 💡 Tilde `~` is a shorthand notation for `Users/your-name/`.
```

If you want to add the current directory to the stack, you would call `pushd` while in that directory:

```bash
~/projects/project_1 pushd
~ ~/projects/project_1
```

> In this case, you are returned to the first directory in your queue.

To check the contents of the directory stack, use `dirs`:

```bash
dirs
~ ~/projects/project_1
```

If you add the -v flag you can include indexes and increase readability:
```bash
dirs -v
0 ~
1 ~/projects/project_1
```

> > The current directory is always the "top" of the directory stack.

Reviewer: ```suggestion
> 💡 The current directory is always at *the top of the directory stack*.
```

~/projects/project_1 dirs -c
```

Or, to undo the last change of directory and go back to the last visited path:

```bash
cd -
```

> To learn more about the directory stack or to view the differences between `popd` and `pushd` vs `cd` in both `bash` and `zsh`, check out the links in the "Learn More" section.

Reviewer: ```suggestion
To learn more about the directory stack or to view the differences between `popd` and `pushd` vs `cd` in both `bash` and `zsh`, check out the links in the *Learn More* section.
```
